pip_test.go

```diff
@@ -24,7 +24,15 @@ type PipCmd struct {
 	Options []string
 }
 
-func TestPipInstall(t *testing.T) {
+func TestPipInstallNativeSyntax(t *testing.T) {
+	testPipInstall(t, false)
+}
+
+func TestPipInstallLegacy(t *testing.T) {
```

Comment:
Let's add ```// Deprecated``` comment for this function, so that we remember to remove it in the future.